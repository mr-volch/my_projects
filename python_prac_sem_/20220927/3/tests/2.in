2,3,4,5
6,7,8,9
5,4,3,2
9,8,7,6
1,1,2,2
2,2,1,1
1,1,2,2
2,2,1,1

