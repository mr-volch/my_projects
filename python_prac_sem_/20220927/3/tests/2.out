22,22,20,20
46,46,44,44
20,20,22,22
44,44,46,46
